modules/sdk-coin-hbar/src/lib/coinTransferBuilder.ts

```diff
@@ -78,67 +70,13 @@ export class CoinTransferBuilder extends TransferBuilder {
   }
 
   // region Transfer fields
-  /** TODO: [BG-51282] Deprecate to and amount once wp has been fixed
-      Currently work for one recipient by using exatcly one of to + amount or send function
-   */
-  /**
-   * Set the destination address where the funds will be sent,
-   * it may take the format `'<shard>.<realm>.<account>'` or `'<account>'`
-   *
-   * @param {string} address - The address to transfer funds to
-   * @returns {TransferBuilder} - The builder with the new parameter set
-   */
-  to(address: string): this {
```

Comment:
Technically... as a result of a stable release, this is a breaking change. Is there any chance between our last `latest` release and whenever we do another `latest` release that these methods can be successfully used by end customers?

Reply:
It seems that these methods can be accessed by the end user through importing `@bitgo/sdk-coin-hbar`

Reply:
Is it expected that they'll be used before the next stable release by end users?

Reply:
If so, we need to add BREAKING CHANGES: <description> to the commit footer.